js/github.js

```diff
@@ -9,10 +9,36 @@ const doesGitHubCookieExist = () => {
   return !!githubCookieStr;
```

Comment:
What is the point of adding two `!!` not operators ?? Can we please correct it to only `githubCookieStr` ?